edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/versioning/ModuleManagementHttpClientVersioned.cs

```diff
@@ -52,6 +52,13 @@ abstract class ModuleManagementHttpClientVersioned
 
         protected ApiVersion Version { get; }
 
+        protected HttpClient GetHttpClient()
+        {
+            // substract 100ms so that httpclient timeouts before TimeoutAfter
+            TimeSpan httpTimeout = this.operationTimeout - TimeSpan.FromMilliseconds(100);
+            return HttpClientHelper.GetHttpClient(this.ManagementUri, httpTimeout > TimeSpan.Zero ? httpTimeout : this.operationTimeout);
```

Comment:
```suggestion
            // substract 100ms so that httpclient timeouts before TimeoutAfter
            TimeSpan httpTimeout = this.operationTimeout > TimeSpan.FromMilliseconds(100) ? this.operationTimeout - TimeSpan.FromMilliseconds(100) : this.operationTimeout;
            return HttpClientHelper.GetHttpClient(this.ManagementUri, httpTimeout);
```

Reply:
Actually, do we still need the TimeoutAfter with this change? Won't the HttpClient always time out? And then we don't need to subtract either?

Reply:
removed timeoutafter